numpy/polynomial/hermite.py

```diff
@@ -1460,10 +1460,11 @@ def hermfit(x, y, deg, rcond=None, full=False, w=None):
     --------
     >>> from numpy.polynomial.hermite import hermfit, hermval
     >>> x = np.linspace(-10, 10)
-    >>> err = np.random.randn(len(x))/10
+    >>> rng = np.random.default_rng(seed=123)
+    >>> err = rng.normal(scale=1./10,size=len(x))
     >>> y = hermval(x, [1, 2, 3]) + err
     >>> hermfit(x, y, 2)
-    array([1.0218, 1.9986, 2.9999]) # may vary
+    array([1.02294967, 2.00016403, 2.99994614])
```

Comment:
Please omit the `seed` and continue to use the `# may vary`. In our [`np.random` documentation](https://numpy.org/doc/stable/reference/random/index.html) we recommend using large seeds and avoid small common seeds like `123`, and we want to demonstrate this practice in all of our examples. Since none of these are actually demonstrating seeding, we prefer to just use `np.random.default_rng()` with no seed.

In the scipy documentation, our documentation generation and doctesting framework will replace an argumentless `np.random.default_rng()` with one with a common large seed to get reproducibility of the example. We might import that practice here at some point, but for now `# may vary` is fine.

Reply:
Done. I like the future plan of using a common large seed. I knew about the recommendation of using large seeds, just wasn't sure removing it from a current example was wanted.  I'll strip small seeds from future PRs as I go. Thanks for the feedback.